.github/workflows/maven.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v[0-9]*'
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 17
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      - name: Build with Maven
+        run: mvn --batch-mode -P jacoco verify
+
+      - name: Run SonarCloud analysis
+        run: >
+          mvn --batch-mode sonar:sonar
+          -Dsonar.host.url=https://sonarcloud.io
+          -Dsonar.organization=gridsuite
+          -Dsonar.projectKey=org.gridsuite:filter
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+      - name: Broadcast update event
+        if: github.ref == 'refs/heads/main'
+        uses: gridsuite/broadcast-event@main
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          event-type: filter_updated
```

Comment:
Hum, I found the repository name "filter" too plain. Filter of what ? it's a resource, a server, a library, etc... ?

Reply:
ping @geofjamg @jonenst @etiennehomer

Reply:
This name was choosen in daily meeting ...